src/ConstantSum/ConstantSumMath.sol

```diff
@@ -50,3 +50,10 @@ function computeSwapDeltaLiquidity(
         return (params.swapFee).mulDivUp(delta, params.price);
     }
 }
+
+/**
+ * @dev Computes the price using the reserve of token X.
+ */
+function computePrice(ConstantSumParams memory params) pure returns (uint256) {
+    return params.price;
+}
```

Comment:
Not sure if this one is needed? I understand that it might be for consistency regarding the other strategies but I think this one could just be an exception.

Reply:
It's not really but makes the api the same between them all. I think general is better than bespoke so i advocate for keeping it.

Reply:
I'm fine with this, but if it lives in the `ConstantSumMath.sol` and it's not invoked anywhere in e.g. solver, or strategy it doesn't really do anything, so I'm not sure what to do with this at the moment.